spec/models/equipment_item_spec.rb

```diff
@@ -13,6 +13,35 @@
     it { is_expected.to validate_presence_of(:name) }
     it { is_expected.to validate_presence_of(:equipment_model) }
 
+    # 2015-11-09: we can't use the shoulda matchers test for scoped uniqueness
+    # due to the lack of a default value for notes - we could potentially
+    # rework our database schema to add a default value but it seems
+    # unnecessary at the moment
+    it 'ensures unique serials scoped to equipment model if it exists' do
+      em1 = FactoryGirl.create(:equipment_model)
+      em2 = FactoryGirl.create(:equipment_model)
+      FactoryGirl.create(:equipment_item, equipment_model: em1, serial: 'a')
+      em1_item2 = FactoryGirl.build(:equipment_item, equipment_model: em1,
```

Comment:
this is perhaps nitpicky, but could these be named more descriptively? like `em1_a` for this one, `em1_nil` for the next, etc